examples/example-gateway/idl/endpoints/bar/bar.thrift

```diff
@@ -34,6 +34,11 @@ struct BarResponse {
     6: required binary binaryField
 }
 
+struct BarRequestRecur {
```

Comment:
to verify this you want to add some tests to `test/endpoints/bar/` that make requests with example JSON payloads and observe that the proxy returns the correct JSON inputs/outputs over HTTP requests.